src/com/facebook/buck/apple/AppleBundle.java

```diff
@@ -1043,15 +1046,19 @@ private void addResourceProcessingSteps(
                 compiledNibPath));
         break;
       case "strings":
-        stepsBuilder.add(
-            new PlutilConvertStep(
-                getBuildTarget(),
-                getProjectFilesystem(),
-                plutil.getEnvironment(resolver),
-                plutil.getCommandPrefix(resolver),
-                PlutilConvertStep.Format.binary1,
-                sourcePath,
-                destinationPath));
+        if (shouldCompileStringsAsBinaryPlists) {
+          stepsBuilder.add(
+              new PlutilConvertStep(
+                  getBuildTarget(),
+                  getProjectFilesystem(),
+                  plutil.getEnvironment(resolver),
+                  plutil.getCommandPrefix(resolver),
+                  PlutilConvertStep.Format.binary1,
+                  sourcePath,
+                  destinationPath));
+        } else {
+          stepsBuilder.add(CopyStep.forFile(getProjectFilesystem(), sourcePath, destinationPath));
```

Comment:
Do you think it would make sense to run the the plist through plutil either way? This way any comments can still be stripped.